src/components/proposal/ProposalTable.tsx

```diff
@@ -149,7 +149,9 @@ const ProposalTable = ({
         actions={[
           (rowData) => {
             const isCallActive = rowData.call?.isActive ?? true;
-            const readOnly = !isCallActive || rowData.submitted;
+            const readOnly =
+              !isCallActive ||
+              (rowData.submitted && rowData.status == 'EDITABLE_SUBMITTED');
```

Comment:
Is the logic here correct? If a proposal has the status EDITABLE_SUBMITTED and is submitted then the right-hand side of the or will be true so the proposal will readOnly. Conversly if the call is still active, the proposal is submitted but is not EDITABLE_SUBMITTED then it will not be readOnly? I think the == be !==.

Reply:
Yes, I think this should be `!==`. Would you be OK to make the change?

Reply:
Yeah will do

Reply:
Oh yes.. It should be !== I will sort that now! Are there any other changes required? I'm looking into the tests at the moment.

Reply:
I have made this change so the proposal test is now passing but the templet tests are still failing. Seems like when you have submitted a proposal and you are reviewing it you can't seem to click on a button.

Reply:
So that button is appearing as read-only? Where is that button? I may have completely missed it before

Reply:
This button is when reviewing a submitting a proposal, it looks at this page in tests to check the proposal has been submitted.